definitions/v4/nzbgeek.yml

```diff
@@ -0,0 +1,149 @@
+---
+id: nzbgeek-yml
+name: NZBgeek (YML)
+description: "NZBgeek is a Private Usenet Indexer"
+language: en-US
+type: private
+redirect: false
+protocol: usenet
+encoding: UTF-8
+links:
+  - https://api.nzbgeek.info
+
+caps:
+  # These can typically be found at {site}/api?t=caps
+  categorymappings:
+    - { id: 1000, cat: Console, desc: Console }
+    - { id: 1010, cat: Console/NDS, desc: Console/NDS }
```

Comment:
You can skip adding desc to all these I believe, it’ll just use standard desc for the mapped cat

Reply:
Trivial to add (all +- semi-automated and parsed from the Caps Call). And the description is not consistent for all.  But I do believe geek is fine